crates/bevy_render/src/texture/image.rs

```diff
@@ -539,10 +539,10 @@ impl Image {
         self.texture_descriptor.size.height
     }
 
-    /// Returns the aspect ratio (height/width) of a 2D image.
+    /// Returns the aspect ratio (width / height) of a 2D image.
     #[inline]
     pub fn aspect_ratio(&self) -> f32 {
```

Comment:
I feel like this should just return an `AspectRatio`.

Reply:
Alright, will get to it straight away.